examples/todo/data/database.js

```diff
@@ -39,8 +39,12 @@ export function getTodo(id) {
   return todosById[id];
 }
 
-export function getTodos() {
-  return todoIdsByUser[VIEWER_ID].map((id) => todosById[id]);
+export function getTodos(status = 'any') {
+  var todos = todoIdsByUser[VIEWER_ID].map((id) => todosById[id]);
```

Comment:
I can do that. Do you want me to change these two: https://github.com/facebook/relay/blob/58e45dbb9fd08215884597171ce072551f344c06/examples/todo/data/database.js#L74-L75 as well?